docs/changes/5.1.1.md

```diff
@@ -12,7 +12,7 @@ Release date: `2021-xx-xx`
 
 ### Direct Transfer
 
-- [NXDRIVE-2](https://jira.nuxeo.com/browse/NXDRIVE-2):
+- [NXDRIVE-2572](https://jira.nuxeo.com/browse/NXDRIVE-2572): Use "AddChildren" permission instead of "ReadWrite" for entitlement of places where one can upload into
```

Comment:
```suggestion
- [NXDRIVE-2572](https://jira.nuxeo.com/browse/NXDRIVE-2572): Use `AddChildren` permission instead of `ReadWrite` for entitlement of places where one can upload into
```